### PR TITLE
Add Ystar 

### PR DESCRIPTION
**Pull request by ystar2020**
Add Ystar logo and info

### CODE REVIEW

**Comment by ystar2020** [comment]
Ystar ERC20 Token is just a mapping currency issued on the ethereum mainnet. Our main currency is issued on yottachain. For details, please refer to the website https://www.yottascan.io/account/ysr.ystar
There are currently 46w holders of coins on yottachain

**Comment by catenocrypt** [comment]
As per our policy, brand new tokens are not included, there is a minimum requirement of 500 holder addresses and 1000 transactions, in order to filter out scam tokens.
See the criteria at (mentioned in the first message by the bot): https://community.trustwallet.com/t/how-to-submit-a-token-logo/3863#how-to-get-approved
This token has below-limit usage yet (176). Putting PR on hold. Please come back to us when all criteria are met.

https://etherscan.io/token/0xd9a947789974bad9be77e45c2b327174a9c59d71

**Comment by catenocrypt** [comment]
The token contract folder should be in checksum format.  I tried to rename, but I have no permission to push to the source repo.  Please rename.

```
0xd9a947789974bad9be77e45c2b327174a9c59d71 => 0xD9A947789974bAD9BE77E45C2b327174A9c59D71
```

**Comment by ystar2020** [comment]
Have been renamed， please check

**Comment by catenocrypt** [comment]
As per our policy, brand new tokens are not included, there is a minimum requirement of 500 holder addresses and 1000 transactions, in order to filter out scam tokens.
See the criteria at (mentioned in the first message by the bot): https://community.trustwallet.com/t/how-to-submit-a-token-logo/3863#how-to-get-approved
This token has below-limit usage yet (188 addresses). Putting PR on hold. Please come back to us when all criteria are met.

https://etherscan.io/token/0xD9A947789974bAD9BE77E45C2b327174A9c59D71

**Comment by ystar2020** [comment]
This is not a brand-new token, it is already listed in Bithum global, Biki and NVEX, daily tranding volume is $500K+, please refer Coinmarketcap and Coingecko https://coinmarketcap.com/currencies/ystar/  https://www.coingecko.com/en/coins/ystar.
YSR is based on YottaChain, it has huge userbase,  more than 460k holders, pleaser refer https://yottascan.io/account/ysr.ystar. YSR-ERC20 is cross-chain mapped from YottaChain for Uniswap only. This is why YSR-ERC20 seems new and very few holder.
Furthermore, YSR is high rank in Uniswap. It is No. 16 now. We already contribute $5M+ liquidity to Uniswap.
In summary, YSR is not spam, please add it. Thanks a lot.

**Comment by ystar2020** [comment]
image

**Comment by ystar2020** [comment]
https://uniswap.info/tokens

**Comment by catenocrypt** [comment]
@ystar2020 , thanks for the clarification, accepted.
